packages/teuchos/comm/src/Teuchos_CommHelpers.hpp

```diff
@@ -254,6 +254,27 @@ scatter (const Packet sendBuf[],
      "See Bug 6375 and Bug 6336.");
 }
 
+template<typename Ordinal, typename Packet>
+void
+scatterv (const Packet sendBuf[],
```

Comment:
There should be some basic native Teuchos unit test for this function.  We don't want code that is only tested in downstream packages.

Reply:
Thank you, @bartlettroscoe. We'll add a unit test to Teuchos.

Reply:
@iyamazaki, those must be bugs from the old Bugzilla site for Trilinos, before Trilinos moved to GitHub (back on 2009?).